README.md

```diff
@@ -1,47 +1,97 @@
-    * Dotenv is a zero-dependency module that loads environment variables from a `.env` file into `process.env`
-
-Development utilities:
-
-* [nodemon](https://www.npmjs.com/package/nodemon)
-    * nodemon is a tool that helps develop node.js based applications by automatically restarting the node application
-      when file changes in the directory are detected.
-* [eslint](https://www.npmjs.com/package/eslint)
-    * ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code.
-* [mocha](https://www.npmjs.com/package/mocha)
-    * ☕️ Simple, flexible, fun JavaScript test framework for Node.js & The Browser ☕️
-* [supertest](https://www.npmjs.com/package/supertest)
-    * HTTP assertions made easy via superagent.
+# Express AEViso API
+
+Team:  
+ -Aeryle: https://github.com/Aeryle  
+ -Chabelle78: https://github.com/Chabelle78  
+ -ThomasBarrial: https://github.com/ThomasBarrial  
+ -MatthiasWanner: https://github.com/MatthiasWanner  
+ -Benwade-progldc: https://github.com/Benwade-prog
+
+API REST made for AEVISO client project: https://github.com/WildCodeSchool/btz-0321-aeviso-client.  
+With jwt authentication.
 
 ## Setup
 
+- run `$ git clone git@github.com:WildCodeSchool/btz-0321-aeviso-api.git` in your terminal
+- run `$ cd btz-0321-aeviso-api`
+- run `npm install `
+
+We use a POSTRGESQL database. If you already have it, paste the url in DATABASE_URL environment variable (in your .env file at the root of the project).  
+If not, a `docker-compose.yml` file can be used:
+
+- Be sure your docker app is running
+- In your terminal, run `docker-compose up -b`
+
+Check if the container is running now with an `adminer` image at `8080` port, and a `postgres` image at `5342` port.  
+Check if the `aeviso` database was created running `http://localhost:8080` in your browser.  
+If all is ok, you should be on the adminer login page. Complete the form like that:
+
+![Adminer](images/adminer_form.png)
+
+password: `aeviso`.
+
+`aeviso` database should appear in the list, but she's empty.
+
+- If done, paste `DATABASE_URL=postgres://aeviso:aeviso@localhost:5432/aeviso` in your .env
+- It will connect the db to the API.
+- Now you can run `npx prisma migrate dev` to create tables and schemas in you db. Confirm message should appear in your console.
+- If you want to push some datas in the db, run `node prisma/seed.js`
+
+## Environment Variables
+
+To run this project, you will need to add the following environment variables to your .env file
+
+`CLIENT=client/url`
+
+`SECRET=secret_key_used_by_jwt`
+
+`DATABASE_URL=your/db/url`
+
+`USER_JOB=jobLabelYouWantToCreateOnServerLaunch`
+
+`USER_FIRSTNAME=firstname`
+
+`USER_LASTNAME=lastname`
+
+`USER_EMAIL=mail@example.dev`
+
+`USER_PASSWORD=password`
+
+## Development
+
 ```
-npm install
+npm run dev
 ```
 
-## Lint
+and run
 
 ```
-npm run lint
+http://localhost:5000
+
 ```
 
-## Test
+## Documentation
+
+go to `/api-docs`
+
+## Facultativs Variables
```

Comment:
Small typo. Faculativs -> Facultatives. Also wouldn't "Optional" be a better name?